src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeFormatter.cs

```diff
@@ -47,6 +47,11 @@ public override string Parameters
             get {return null;}
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
It's probably the "small numbers" case.  An EncryptedXml object is the only netfx user of these objects (well, some by SignedXml); and this particular class is used by 0.2% (2/1000) of applications-in-the-wild.

This particular property has the fun stat of showing as 0 usage.

https://apisof.net/catalog/System.Security.Cryptography.RSAPKCS1KeyExchangeDeformatter.RNG

So, sure, let's have the extra field.